python/pyspark/tests/test_worker.py

```diff
@@ -175,13 +175,12 @@ def test_reuse_worker_of_parallelize_xrange(self):
         for pid in current_pids:
             self.assertTrue(pid in previous_pids)
 
-
```

Comment:
I wouldn't change such newlines BTW. This is per pep8. another newline below, I don't think that violates pep 8.

Reply:
My bad. linter doesn't work in my machine (maybe it's because the the default python in my dev. is 2.7?) so missed that. Will revert.